src/Microsoft.DotNet.MSBuildSdkResolver/FXVersion.cs

```diff
@@ -86,6 +172,10 @@ public static bool TryParse(string fxVersionString, out FXVersion FXVersion)
             {
                 return false;
             }
+            if (minorSeparator - minorStart > 1 && fxVersionString.Substring(minorStart, 1) == "0")
```

Comment:
Also applies to checks for single chars in the middel of strings. This can be `fxVersionString[minorStart] == '0'`